### PR TITLE
UI: Add Back Button To Wallet Creation

### PR DESCRIPTION
**Pull request by dills122**
# Issue
During initial setup, importing wallet views had a back button that would navigate the user to the `Add Wallet` view, but the `Create Password` view during wallet creation did not have this setup.

# Fix
Add a back button that is in line with the look of the `Add Wallet` and other setup views.

### CODE REVIEW

**Review by Shadowfiend — CHANGES_REQUESTED** [review]
Thanks for the contribution! Small note below, and I want to give it a test drive and make sure no unexpected state ends up in the system, but this seems like a great little UX addition :)

File: ui/components/Keyring/KeyringSetPassword.tsx [hunk 1]
```diff
@@ -96,6 +100,20 @@ export default function KeyringSetPassword(): ReactElement {
...
+          .wordmark {
+            background: url("./images/wordmark@2x.png");
+            background-size: cover;
+            width: 52px;
+            height: 25px;
+            position: absolute;
+            left: 0px;
+            right: 0px;
+            margin: 0 auto;
+          }
```
[the remark this answers is not in the document]
Comment (dills122) [reply ui/components/Keyring/KeyringSetPassword.tsx:116]: Got this updated, combined the two rulesets.

**Comment by dills122** [comment]
No problem! Hoping to try make it a more regular thing and get more involved with the project.

Really digging this project!

**Review by cairomassimo — APPROVED** [review]
Looks good!

**Comment by cairomassimo** [comment]
Thanks @Shadowfiend for merging! I was waiting for CI then forgot to do it (and to enable auto-merge).

**Pull request merged**